recipes/openassetio/all/conanfile.py

```diff
@@ -0,0 +1,165 @@
+import pathlib
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.apple import is_apple_os
+from conan.tools.files import apply_conandata_patches, get, copy, rm
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.env import VirtualBuildEnv
+
+
+required_conan_version = ">=1.53.0"
+
+
+class PackageConan(ConanFile):
+    name = "openassetio"
+    description = "An open-source interoperability standard for tools and content management systems used in media production."
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/OpenAssetIO/OpenAssetIO"
+    topics = ("asset-pipeline", "vfx", "cg", "assetmanager", "vfx-pipeline")
+    settings = "os", "arch", "compiler", "build_type"
```

Comment:
pretty sure this is a lib not an application but correct me if I am wrong

```suggestion
    package_type = "library"
    settings = "os", "arch", "compiler", "build_type"
```

Reply:
Another case of the `package_templates` changing under our feet.